### ISSUE TITLE
Remove or reduce "The sudden rush of air knocks you over!"

### ISSUE THREAD

**Issue by PJB3005** [d3athrow/vgstation13#24970]
**NOTE:** I am not calling for removal of airflow throwing you around, that's totally fine.

What I *do* hate is walking through hallways, having a random "the sudden rush of air knocks you over!" because somebody opened a door on the other end of the station, and getting knocked over for 10 seconds straight.

**Comment by Shadowmech88** [comment]
Is there any way to detect when someone is blown into a wall as opposed to simply being knocked over?
Getting thrown across the room is deserving of such a long stun, getting swept off your feet really isn't. Slips don't stun for that long.

**Comment by SonixApache** [comment]
getting knocked over when you don't even get moved one tile is shitty

**Comment by PJB3005** [comment]
> Is there any way to detect when someone is blown into a wall as opposed to simply being knocked over?

Yes, there are completely separate code paths for a single stun vs getting flung around the station.

> getting knocked over when you don't even get moved one tile is shitty

Yep.

**Comment by utilitas-** [comment]
I don't care about getting knocked from halfway across the station. That's them hardcores.
What I do care about is the occasional and complete stunlock by constantly changing airflow, like, say, in a fairly small and restricted high-pressure room that suddenly gets spaced. If it won't kill you, it will certainly take you down to orange health and take you out of commission with you being able to do not a single thing about it, not even turn on your magboots.

**Comment by DeityLink** [comment]
airflow stun now depends on the actual pressure differential, I don't know if this was the case back then.
